M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -653,13 +653,15 @@ doc ///
 	     L = randomAbstractSimplicialComplex(6,3)
 	  Text
 	     Create the random complex $Y_d(n,m)$ which has vertex set
-             $[n]$ and complete $(d − 1)$-skeleton, and has exactly m d-dimensional faces,
+             $[n]$ and complete $(d − 1)$-skeleton, and has exactly $m$ $d$-dimensional faces,
```

Comment:
You should either remove the word "exactly" here, or alternatively after you create a random simplicial complex check that it meets both conditions and otherwise try again.